ios/Podfile.lock

```diff
@@ -638,4 +638,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ca4eb736278d4211ffaa5cef62299ef8fc2137ca
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.11.3
```

Comment:
Well after library updates, always someone should apply an updated podfile.lock. I guess this change is due to the other library update tickets. I would keep this update, since it has to be done because of deprecation warnings while doing `pod install`

Reply:
reverted